include/nuttx/net/mii.h

```diff
@@ -396,6 +396,13 @@
 #define DP83840_PHYADDR_DUPLEX       (1 << 7)
 #define DP83840_PHYADDR_SPEED        (1 << 6)
 
+/* Davicom DM9161  **********************************************************/
+
+/* DM9161  MII ID1/2 register bits */
+
+#define MII_PHYID1_DM9161 			0x0181
```

Comment:
```suggestion
#define MII_PHYID1_DM9161            0x0181
```
change tab to space and algin with line 397